.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,23 @@
-_Motivation:_  Explain here what the reason for the pull request is.
-_Linked issues:_  Optionally, add links to existing issues or other PR's
+## Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.
+
+Fixes # Optionally, add links to existing issues or other PR's
+
+## Checklist
 
-### Checklist
-- [ ] Build rule `all-supported` completed successfully
```

Comment:
I would like to keep the Build ... completed... Check to motivate contributors to build all-supported before creating a PR.

Reply:
I see your point and added it back, but I haven't done it myself for quite a while. The CI does this mostly for us anyway.